src/main/java/uk/gov/hmcts/reform/sendletter/entity/Letter.java

```diff
@@ -43,6 +46,8 @@
     private String type;
     @Enumerated(EnumType.STRING)
     private LetterStatus status = LetterStatus.Created;
+
+    @Basic(fetch = LAZY)
     private byte[] pdf;
```

Comment:
care to explain?

Reply:
all hibernate queries are eagerly loaded by default. to avoid fetching pdf whenever we query for `Letter` - set `basic` usage of a resource fetch